src/App.tsx

```diff
@@ -1,14 +1,66 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, {
+  useContext, useEffect, useMemo, useState,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import './app.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import { TodoContext } from './components/TodoContext/TodoContext';
 
 export const App: React.FC = () => {
+  const {
+    activeTodo,
+    setActiveTodo,
+    query,
+    filter,
+  } = useContext(TodoContext);
+
+  const [loading, setLoading] = useState(false);
```

Comment:
Rename all boolean variables according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig). Fix all cases
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```